clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -6003,6 +6003,11 @@ RValue CodeGenFunction::EmitCall(QualType CalleeType, const CGCallee &OrigCallee
     }
   }
 
+  // Set type identifier metadata of indirect calls for call graph section.
+  if (CGM.getCodeGenOpts().CallGraphSection && CallOrInvoke &&
+      CallOrInvoke->isIndirectCall())
```

Comment:
Is `CallOrInvoke` in scope? it appears to be defined in the block above. If its shadowing, maybe we want to change that or make sure we don't want the shadowed value (e.g. the now out of scope def from line 5990.

Reply:
Verified that it is in scope.